lib/methods/pluginnotificationtask.cpp

```diff
@@ -48,8 +63,35 @@ void PluginNotificationTask::ScriptFunc(const Notification::Ptr& notification,
 	resolvers.emplace_back("user", user);
 	resolvers.emplace_back("notification", notificationExtra);
 	resolvers.emplace_back("notification", notification);
-	if (service)
+
+	if (service) {
+#ifdef __linux__
+		auto cr (service->GetLastCheckResult());
+
+		if (cr) {
+			auto output (cr->GetOutput());
+
+			if (output.GetLength() > l_MaxOutLen) {
+				resolvers.emplace_back("service", new Dictionary({{"output", output.SubStr(0, l_MaxOutLen)}}));
```

Comment:
If it's Linux, the macro resolver key `service` is set twice? Is it safe to rely on the vector element insertion order and expect this element to be found before the actual service object?

Reply:
_notification_ is already set twice, on all platforms.